### PR TITLE
Change .puml files to match requirement

### PR DESCRIPTION
**Pull request by reetmitra**

### CODE REVIEW

File: docs/diagrams/BetterModelClassDiagram.puml [hunk 1]
```diff
@@ -14,8 +14,8 @@ UniquePersonList -right-> Person
 
 Person -up-> "*" Tag
 
-Person *--> Name
 Person *--> Email
+Person *--> Name
```
Comment (Hibeom0929) [inline docs/diagrams/BetterModelClassDiagram.puml:18]: I think Name should be before Email

File: docs/diagrams/ModelClassDiagram.puml [hunk 2]
```diff
@@ -51,8 +51,8 @@ Person *--> Role
 Person -[hidden]up--> I
 UniquePersonList -[hidden]right-> I
 
-Name -[hidden]right-> Phone
 Phone -[hidden]right-> Address
+Name -[hidden]right-> Phone
```
Comment (Hibeom0929) [inline docs/diagrams/ModelClassDiagram.puml:55]: Same here.

**Comment by E0735389** [comment]
What's the requirement? Why are we doing this?

File: docs/diagrams/BetterModelClassDiagram.puml (hunk 1 above)
Comment (reetmitra) [reply docs/diagrams/BetterModelClassDiagram.puml:18]: Oh i was just changing the lines for the UML diagram tracker thing right my bad

Comment (E0735389) [reply docs/diagrams/BetterModelClassDiagram.puml:18]: This is cheating but... I guess we will ignore it.

**Pull request merged**